app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -51,17 +50,14 @@ class HecTaxCheckExtractionServiceImpl @Inject() (
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
-  val generateLock: LockKeeper =
-    new LockKeeper {
-
-      override def repo: LockRepository            = LockMongoRepository(mongo.mongoConnector.db)
-      override def lockId: String                  = "hecTaxChecks"
-      override val forceLockReleaseAfter: Duration =
-        Duration.standardMinutes(config.get[Long]("mongo-lock.forceLockReleaseAfter"))
-    }
+  val lockService: LockService = LockService(
+    mongoLockRepository,
+    lockId = "hecTaxChecks",
+    ttl = Duration(config.get[Long]("mongo-lock.forceLockReleaseAfter"), MINUTES)
```

Comment:
didn't pick this up before, but it'd be worth making it clear what units the config is in in the name like `forceLockReleaseAfterMinutes` or something so that we don't have to wonder if we're just looking at the config in isolation. Also, we've got kebab case in the rest of the config so it'd be neater to have this config use the same, i.e. `force-lock-release-after-minutes`

Reply:
updated